packages/migration-graph-shared/src/entities/project-graph.ts

```diff
@@ -0,0 +1,65 @@
+import debug from 'debug';
+import { Graph, GraphNode } from '../graph';
+import { Package } from './package';
+
+import type { ModuleNode, PackageNode } from '../types';
+
+const DEBUG_CALLBACK = debug('rehearsal:project-graph');
+
+// TODO this package level dependency data should be surfaced in a report
```

Comment:
@rehearsal/cli#migrate will be exposing a report containing migration tasks which are scoped in order from migration-graph. that should suffice.